src/utils.ts

```diff
@@ -64,16 +62,26 @@ export function getDirs() {
   };
 }
 
-export function getAppConfig() {
-  const appSecret = process.env.APP_SECRET?.trim() ?? uuidv4();
-  const appId = createHash("sha256").update(appSecret).digest("hex");
-  return { appId, appSecret };
+export async function getAppConfig() {
+  const res = await axios.post<{ appId: string; appSecret: string }>("https://coordinator.hathora.dev/registerApp");
+  return res.data;
 }
 
-export function generateLocal() {
+export async function generateLocal() {
   const { rootDir } = getDirs();
-  dotenv.config({ path: join(rootDir, ".env") });
-  const appConfig = getAppConfig();
+
+  let appConfig: { appId: string; appSecret: string };
+  if (!existsSync(join(rootDir, ".env"))) {
+    appConfig = await getAppConfig();
+    outputFileSync(join(rootDir, ".env"), `APP_ID=${appConfig.appId}\nAPP_SECRET=${appConfig.appSecret}\n`);
```

Comment:
parse this, check if file doesn't exist or APP_ID isn't present, generate a new one